### DISCUSSION TITLE
How would I make it so that slash command responses could be edited?

### DISCUSSION THREAD

**Discussion by DumbLuckGames** [Rapptz/discord.py#9101] [Q&A] [answered]
Currently I use 
`message = await interaction.response.send_message("Content blurb")`

`await message.edit(content = "edited blurb")` 

Does anyone know how I could make this work so that there isn't an error given?

**Comment by Rapptz** [comment, marked as answer]
Interaction response endpoint doesn't return anything.

To do this you'll have to call [`Interaction.edit_original_response`](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.Interaction.edit_original_response) to edit the message you responded with.

**Reply by DumbLuckGames** [reply]
It worked! thank you so much.

**Comment by Awakno** [comment]
don't work for me 
`   await message.edit(content = "edited blurb")
AttributeError: 'Interaction' object has no attribute 'edit'`

**Reply by MadCat9958** [reply]
`await message.edit_original_response` instead of `await message.edit`
